.env

```diff
@@ -0,0 +1,2 @@
+seed=256a818b2aac458941f7274985a410e57fb750f3a3a67969ece5bd9ae7eef5b2
```

Comment:
To remove this from Git, run `git rm -r --cached .`, and then `git add .` and `git commit`